source/simulator/assembly.cc

```diff
@@ -233,6 +249,7 @@ namespace aspect
           current_strain_rates(quadrature.size(), Utilities::signaling_nan<SymmetricTensor<2,dim> >()),
           current_composition_values(n_compositional_fields,
                                      std::vector<double>(quadrature.size(), Utilities::signaling_nan<double>())),
+          current_velocity_divergences(quadrature.size()),
```

Comment:
Good catch! We actually found a place where we used it without filling in the values first...